libass/ass_cache.c

```diff
@@ -99,7 +99,7 @@ const CacheDesc bitmap_cache_desc = {
 
 
 // composite cache
-static uint32_t composite_hash(void *key, uint32_t hval)
+static uint64_t composite_hash(void *key, uint64_t hval)
```

Comment:
I think it's better to use `typedef uint64_t ass_hash_t;` and use it. First commit looks like the perfect place to do it.

Reply:
@rcombs Any comment on the suggestion to introduce an `ass_hash_t`?

Reply:
`_t` names are reserved for the system libraries as per POSIX and defining types with such names is undefined behavior, so I'm not going to add a new one of those. We could maybe do `ass_hashcode`?

Reply:
Right, of course; but do you like the idea? It seems pretty sensible to me.